### ISSUE TITLE
Search Tab

### ISSUE THREAD

**Issue by madebr** [madebr/subdownloader#13]
Show folders in alphabetical order.

**Comment by sergiomb2** [comment]
yes , I also notice window tree and other sometimes are not in alphabetical order. 
Also an RFE is user may choose how is it sorted ...
